modules/caddypki/acmeserver/acmeserver.go

```diff
@@ -160,11 +158,64 @@ func (ash Handler) ServeHTTP(w http.ResponseWriter, r *http.Request, next caddyh
 	return next.ServeHTTP(w, r)
 }
 
+func (ash Handler) getDatabaseKey() string {
+	return ash.CA
+}
+
+// Cleanup implements caddy.CleanerUpper and closes any idle databases.
+func (ash Handler) Cleanup() error {
+	key := ash.getDatabaseKey()
+	deleted, err := databasePool.Delete(key)
+	if deleted {
+		ash.logger.Debug("unloading unused database", zap.String("caID", key))
```

Comment:
```suggestion
		ash.logger.Debug("unloading unused database", zap.String("db_key", key))
```

(We don't know that it's the CA ID since that is abstracted away into `getDatabaseKey()`)